test/integration_tests/test_select_executor.py

```diff
@@ -60,7 +60,7 @@ def test_should_load_and_select_in_table(self):
         expected_batch = list(create_dummy_batches())
         self.assertEqual([actual_batch], expected_batch)
 
-    @unittest.skip('Too slow when batch size is small')
+    @unittest.skip('Too slow when batch size is small.')
```

Comment:
Is this slow for batch size 50? How about we set the configuration before running this test case and reset it back?

Reply:
30 seconds for batch size 50. We can keep batch size 50. There is no problem for that.

Reply:
In test_pytorch, we will do this again. So I think it maybe better to disable this one for saving time.

Reply:
Makes sense